### PR TITLE
CASH-704 Convert GridLoanCard to use LoanCardController

### PR DESCRIPTION
**Pull request by foochris**

### CODE REVIEW

File: src/pages/Lend/LendByCategoryPage.vue [hunk 1]
```diff
@@ -413,6 +413,8 @@ export default {
 			this.showFeaturedHeroLoan = true;
 		}
 
+		this.showFeaturedLoans = true;
```
Comment (mcstover) [inline src/pages/Lend/LendByCategoryPage.vue]: For local testing I presume...

**Comment by mcstover** [comment]
@foochris Looking good so far!

File: src/pages/Lend/LendByCategoryPage.vue (hunk 1 above)
Comment (foochris) [reply src/pages/Lend/LendByCategoryPage.vue]: @mcstover Yup, WIP. 'Draft' PR. :)

**Comment by foochris** [comment]
@mcstover Does Github notify if the PR changes from Draft to Ready for review? If not, this PR is ready for review..

**Pull request merged**